vms/platformvm/vm_regression_test.go

```diff
@@ -47,11 +47,12 @@ import (
 	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
 
 	blockexecutor "github.com/ava-labs/avalanchego/vms/platformvm/block/executor"
+	txbuilder "github.com/ava-labs/avalanchego/vms/platformvm/txs/builder"
 )
 
 func TestAddDelegatorTxOverDelegatedRegression(t *testing.T) {
 	require := require.New(t)
-	vm, _, _ := defaultVM(t, cortina)
+	vm, txBuilder, _, _ := defaultVM(t, cortina)
```

Comment:
this is the one bit slightly annoying to me: why `defaultVM` should return the `txBuilder`?
Can we instead create the `txBuilder` only in the tests where we need it and leave defaultVM signature unchanged?

Reply:
over 50% of the tests use txBuilder (including `defaultVM` itself). I think we should keep it in there